src/test/java/org/springframework/data/elasticsearch/repository/query/keywords/QueryKeywordsTests.java

```diff
@@ -155,6 +158,40 @@ public void shouldSupportLessThanAndGreaterThan() {
 		assertThat(repository.findByPriceGreaterThanEqual(1.9f)).hasSize(2);
 	}
 
+	@Test // DATAES-615
+	public void shouldSupportSortOnStandardFieldWithCriteria() {
+		final List<String> sortedIds = repository.findAllByNameOrderByText("Salt").stream() //
```

Comment:
Nit: No need for `final` keyword in local variables.

Reply:
that's IJ on refcatoring - must check where to disable that